src/bp-groups/bp-groups-template.php

```diff
@@ -429,11 +429,6 @@ function bp_has_groups( $args = '' ) {
 		$parent_id = 0;
 	}
 
-	$show_hidden = true;
-	if ( ! empty( bp_displayed_user_id() ) && ! empty( bp_current_user_id() ) && bp_displayed_user_id() === bp_current_user_id() ) {
-		$show_hidden = false;
-	}
-
```

Comment:
it will reopen the issue which fixed here: https://github.com/buddyboss/buddyboss-platform/pull/2175